AudioBar/AudioBarViewController.swift

```diff
@@ -96,6 +108,43 @@ public final class AudioBarViewController: UIViewController, StoreDelegate {
         }
     }
 
+    private func configureCommandCenterCommands() {
+        commandCenter.playCommand.addTarget { [weak store] _ in
```

Comment:
These commands are ridiculously complicated! We said no logic in the imperative shell and here we are.

All we want to do here is:

```swift
store!.dispatch(.remoteCommand(.userDidTapPlayButton))
```